src/main/webapp/WEB-INF/views/allItems.jsp

```diff
@@ -0,0 +1,66 @@
+<%@ taglib uri = "http://java.sun.com/jsp/jstl/core" prefix = "c" %>
+<jsp:useBean id="items" scope="request" type="java.util.List<mateacademy.internetshop.model.Item>"/>
+<%--
+  Created by IntelliJ IDEA.
+  User: andre
+  Date: 17-Sep-19
+  Time: 18:24
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>AllItems</title>
+
+    <style>
```

Comment:
optional: better move style in  the css file

Reply:
I just want to make it well looking...